src/main/java/com/jalasoft/jfc/model/VideoParam.java

```diff
@@ -0,0 +1,294 @@
+    private String rotate;          //Content rotate value
+
+    /**
+     * Constructor of VideoParam class.
+     * @param fFmpeg is for getting the binary path value.
+     */
+    public VideoParam(String fFmpeg){
+        this.fFmpeg = fFmpeg;
+    }
+
+    /**
+     * getfFmpeg gets fFmpeg value.
+     * @return fFmpeg String value.
+     */
+    public String getfFmpeg() {
+        return fFmpeg;
+    }
+
+    /**
+     * getInputPathFile gets inputPathFile value.
+     * @return inputPathFile String value.
+     */
+    public String getInputPathFile() {
+        return inputPathFile;
+    }
+
+    /**
+     * getInputPathFile gets inputPathFile value.
+     * @return inputPathFile String value.
+     */
+    public String getOutputPathFile() {
+        return outputPathFile;
+    }
+
+    /**
+     * getOutputFileName gets outputFileName value.
+     * @return outputFileName String value.
+     */
+    public String getOutputFileName() {
+        return outputFileName;
+    }
+
+    /**
+     * getAspectRatio gets aspectRatio value.
+     * @return aspectRatio double value.
+     */
+    public double getAspectRatio() {
+        return aspectRatio;
+    }
+
+    /**
+     * getFrameRate gets frameRate value.
+     * @return frameRate String value.
+     */
+    public String getFrameRate() {
+        return frameRate;
+    }
+
+    /**
+     * getWight gets wight value.
+     * @return wight int value.
+     */
+    public int getWight() {
+        return wight;
+    }
+
+    /**
+     * getHeight gets height value.
+     * @return height int value.
+     */
+    public int getHeight() {
+        return height;
+    }
+
+    /**
+     * getVideoCodec gets videoCodec value.
+     * @return videoCodec String value.
+     */
+    public String getVideoCodec() {
+        return videoCodec;
+    }
+
+    /**
+     * getAudioCodec gets audioCodec value.
+     * @return audioCodec String value.
+     */
+    public String getAudioCodec() {
+        return audioCodec;
+    }
+
+    /**
+     * getVideoBitRate gets videoBitRate value.
+     * @return videoBitRate String value.
+     */
+    public String getVideoBitRate() {
+        return videoBitRate;
+    }
+
+    /**
+     * getAudioBitRate gets audioBitRate value.
+     * @return audioBitRate String value.
+     */
+    public String getAudioBitRate() {
+        return audioBitRate;
+    }
+
+    /**
+     * getQuality gets quality value.
+     * @return quality byte value.
+     */
+    public byte getQuality() {
+        return quality;
+    }
+
+    /**
+     * getChannelsNumber gets channelsNumber value.
+     * @return channelsNumber byte value.
+     */
+    public byte getChannelsNumber() {
+        return channelsNumber;
+    }
+
+    /**
+     * getVolume gets volume value.
+     * @return volume String value.
+     */
+    public String getVolume() {
+        return volume;
+    }
+
+    /**
+     * getRotate gets rotate value.
+     * @return rotate String value.
```

Comment:
Would be more especific the documentation of all return like "degrees of rotation"